src/main/resources/filing/templates/small-full-accounts.html

```diff
@@ -0,0 +1,1503 @@
+{{"<?xml version=\"1.0\" encoding=\"UTF-8\"?>" | unescaped}}
+
+<html xmlns:ixt2="http://www.xbrl.org/inlineXBRL/transformation/2011-07-31" xmlns="http://www.w3.org/1999/xhtml" xmlns:core="http://xbrl.frc.org.uk/fr/2014-09-01/core" xmlns:xbrldi="http://xbrl.org/2006/xbrldi" xmlns:curr="http://xbrl.frc.org.uk/cd/2014-09-01/currencies" xmlns:countries="http://xbrl.frc.org.uk/cd/2014-09-01/countries" xmlns:uk-bus="http://xbrl.frc.org.uk/cd/2014-09-01/business" xmlns:ix="http://www.xbrl.org/2013/inlineXBRL" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:xbrli="http://www.xbrl.org/2003/instance" xmlns:direp="http://xbrl.frc.org.uk/reports/2014-09-01/direp" xmlns:xlink="http://www.w3.org/1999/xlink" xml:lang="en" xmlns:link="http://www.xbrl.org/2003/linkbase" xmlns:iso4217="http://www.xbrl.org/2003/iso4217">
+  {{$company := .small_full_account.company}}
+  {{$companyName := $company.company_name}}
+  {{$companyNumber := $company.company_number}}
+  <head>
+    <meta content="application/xhtml+xml; charset=UTF-8" http-equiv="content-type" />
+    <title>
+      {{$companyName}}
+    </title>
+    <style type="text/css">
+
+body {
```

Comment:
The CSS present in the example small full iXBRL differs from what is currently present (I think this CSS may relate to abridged). This may cause issues down the line. Recommend replacing the CSS within the document with the CSS present in the example.